server/src/com/cloud/api/query/QueryManagerImpl.java

```diff
@@ -1731,6 +1731,17 @@
         Long zoneId = cmd.getZoneId();
         Long podId = cmd.getPodId();
 
+        List<Long> ids = null;
```

Comment:
Ok, they extend from <code>com.cloud.utils.component.ManagerBase</code> but I found an issue on that, from its project (cloud-utils) there is no visibility to <code>com.cloud.exception.InvalidParameterValueException</code>
I wouldn't like to modify dependencies, what do you think? Would I use another exception?